tests/light/src/common/network.py

```
@@ -48,7 +48,10 @@ async def stop(self):
 
     async def close_client(self):
         """After a client connection is accepted, new connections will be rejected until this method is called"""
-        await self._client.close()
+        try:
+            await self._client.close()
+        except ConnectionResetError:  # remove this after fixing two way shutdown (SSL_shutdown) in AxoSyslog
+            pass
```

This part will silently pass for the testcase, can not we add a `logger.warning("some info about the exception")` until it is not fixed in Axosyslog?

---

For example,
`client closed connection prematurely`

---

#420 will fix the issue and we can remove the `try-except`. Let's see which PR goes in first.

---

Added warning message.

---

Thanks. After `logger.warning()` the `pass` expression does not need anymore

---

Removed.